lib/cli/commands/removeorder.ts

```diff
@@ -1,7 +1,17 @@
 import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
-import { RemoveOrderRequest } from '../../proto/xudrpc_pb';
+import { RemoveOrderRequest, RemoveOrderResponse } from '../../proto/xudrpc_pb';
 import { coinsToSats } from '../utils';
+import Table, { VerticalTable } from 'cli-table3';
+import colors from 'colors/safe';
+
+const displayRemoveOrder = (res: RemoveOrderResponse.AsObject) => {
```

Comment:
Per my earlier comment:

> I actually think the table makes this less readable. How about just printing a message like "Order removed" if quantity on hold is zero or "Order removed, X quantity is on hold pending a swap" otherwise?

I'd be fine with just reverting changes to this file for now and take care of this change in a separate PR, but I still think a table makes no sense here.